assets/js/components/Dashboard/PermanentConfiguration/Workflows/GlEink/GlEinkWorkflow.tsx

```diff
@@ -247,6 +259,35 @@ const GlEinkWorkflow: ComponentType<WorkflowProps> = ({
       };
       layout = (
         <>
+          <Modal
+            show={showErrorModal}
+            backdrop="static"
+            className="error-modal"
+          >
+            <Modal.Header closeButton closeVariant="white">
```

Comment:
I think we need to `setShowErrorModal(false)` when the close button is pressed as well